python/pylibcudf/pylibcudf/strings/convert/convert_durations.pyx

```diff
@@ -19,7 +20,7 @@ cpdef Column to_durations(
     Column input,
     DataType duration_type,
     str format,
-    Stream stream=None
+    Stream stream=None, DeviceMemoryResource mr=None
```

Comment:
Nit: Odd that the `mr`s in this file weren't placed on a new line

Reply:
Sometimes assistants randomly do different things, good eye. Fixed.